src/main/java/no/digipost/api/client/representations/MessageStatus.java

```diff
@@ -25,6 +25,7 @@
     NOT_COMPLETE,
     COMPLETE,
     DELIVERED,
+    DELIVERED_WITH_PRINT_FALLBACK,
```

Comment:
I see this is only a draft pull request, but is this new status necessary? If you send a message with a `print-fallback-deadline`, and say you get a `DELIVERED` status back, does that not imply that everything was ok, the message was delivered as requested _with_ the the print fallback?

Reply:
Well, on one hand this might make sense, on the other hand we already have a `DELIVERED_TO_PRINT` status as well as a `DELIVERED` status that is used when a document is delivered directly through the printing service. We could of course consider not adding this extra status, I can check with the others 👍

Reply:
We decided to remove it, as per your feedback